Regular-Question-Answers/power-of-three.cpp

```diff
@@ -0,0 +1,23 @@
+// Author: Rojen Arda Şeşen
+// Question Link: https://leetcode.com/problems/power-of-three/
+
+// n is a power of k if log(n) base k  is an integer.
+// We will be using double precision floating point numbers to calculate logarithms.
+// There may be errors in floating point representation, therefore we will check
+// equality based on a treshold.
+
+// Time complexity of this code is O(1), assuming the time complexity of log function
+// is O(1). Space complexity is also O(1).
+
+class Solution {
+public:
+    bool isPowerOfThree(int n) {
+		double result;
+		result = log(n) / log(3);
```

Comment:
You can add the theorem that you used, it will be more explanatory. Other than that it is perfect.